### DISCUSSION TITLE
前端缓存策略如果遇到后端数据更改怎么办

### DISCUSSION THREAD

**Discussion by wintsa123** [alovajs/alova#616] [General]
假设A/b共同在看一个页面，b修改了，a在重新请求，这时候命中的缓存是之前的数据。并非新数据。这种情况怎么解决，后端无法发送信息给a通知清除缓存。

**Comment by JOU-amjs** [comment]
根据数据的场景来决定是否使用缓存，以及缓存的有效时间。

一般而言，对于实时性要求较高的数据不建议缓存，为了短暂提高响应性能可以考虑短时间缓存，例如10秒，时效性低的数据可以设置更高的缓存。
